include/mprio.h

```diff
@@ -93,14 +93,36 @@ void Prio_clear();
 PrioConfig PrioConfig_new(int nFields, PublicKey serverA, PublicKey serverB,
                           const unsigned char* batchId,
                           unsigned int batchIdLen);
+
+/*
+ * Wraps PrioConfig_new. Returns PrioConfig for nInts integers with
+ * precision prec.
+ *
+ * NOTE: For compatibility reasons, prec is not contained in the
+ * PrioConfig struct and must be set client- and serverside.
+ */
+PrioConfig PrioConfig_new_int(int nInts, int prec, PublicKey serverA,
+                              PublicKey serverB, const unsigned char* batchId,
+                              unsigned int batchIdLen);
+
 void PrioConfig_clear(PrioConfig cfg);
 int PrioConfig_numDataFields(const_PrioConfig cfg);
 
+/*
+ * Return the number of integers with precision prec.
```

Comment:
Maybe instead "Return the number of integers with precision prec that this configuration supports."